### PR TITLE
Update on Boombats in Vanilla Animals Expanded - Australia

### PR DESCRIPTION
**Pull request by feeddanoob**
## Additions
Added FSX production to the boombats. and labeled them as boom for trading. Amounts and time for fsx copied from boomrats.

## Reasoning
Boombats also go boom, they could be farmed for chemfuel, and sometimes they overrun the boomrats in the temperate forest biomes. Balancing wise I don't know how much fsx they should be milked/butchered, I thought it was fair to put those amounts the same as boomrats.

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors
- [x] (For compatibility patches) ...with and without patched mod loaded
- [x] Playtested a colony for 4 in game days to see if they can be shaved off of fsx. Got the boombats on day 1.

### CODE REVIEW

File: Patches/Vanilla Animals Expanded/Australia_Animals.xml [hunk 1]
```diff
@@ -10,6 +10,13 @@
       <operations>
         <!-- === Boombat === -->
         <li Class="PatchOperationAdd">
+			<xpath>Defs/ThingDef[defName="Boomalope"]/tradeTags</xpath>
```
Comment (N7Huntsman) [inline Patches/Vanilla Animals Expanded/Australia_Animals.xml:13]: This this supposed to be `AEXP_Boombat` here?

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
Generally looks good, but one possibly typo for you to look at.

File: Patches/Vanilla Animals Expanded/Australia_Animals.xml (hunk 1 above)
Comment (feeddanoob) [reply Patches/Vanilla Animals Expanded/Australia_Animals.xml]: yes... I knew i missed something

**Comment by feeddanoob** [comment]
...The two recent commits were me trying to update the fork

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
You've got some older revisions in here from Genetic Rim that need to go away.

**Comment by feeddanoob** [comment]
How about now?...

**Comment by N7Huntsman** [comment]
I'm closing this, as it's got some other content. I'll submit the VAE changes separately.

**Pull request closed without merging**